index.js

```diff
@@ -644,6 +990,21 @@ module.exports.getBatteryCharge = function(sid, ain, options)
     });
 };
 
+/* The above should be replaced by this, even though the whole device list is queried
+under the hood:
+
+module.exports.getBatteryCharge = function(sid, ain, options)
+{
+    return module.exports.getDevice(sid, ain, options).then(function(device) {
+        return device.battery;
+    });
+};
+
+*/
+
+// Same here as for getBatteryCharge. 
```

Comment:
agreed, could you implement the replacement?

Reply:
Done